seeds/masterSeed.js

```diff
@@ -0,0 +1,23 @@
+// var seed = require('seeds/1seedDescription.js');
+const description = require('./1seedDescription.js');
+const attributes = require('./2seedAttributes.js');
+const details = require('./3seedDetails.js');
+const directions = require('./4seedDirections.js');
+
+exports.seed = function (knex, Promise) {
+  return description
+    .seed(knex, Promise)
+    .then(function () {
+      return details.seed(knex, Promise);
```

Comment:
is this supposed to be descriptions?

Reply:
NO I named it description.

Reply:
But i'm gonna change it actually.

Reply:
o wait i thought i saw details twice my bad

Reply:
lol no worries. i changed it anyways, for uniformity